tidy3d/components/simulation.py

```diff
@@ -3245,6 +3249,24 @@ def _post_init_validators(self) -> None:
         self._validate_tfsf_nonuniform_grid()
         self._validate_nonlinear_specs()
         self._validate_custom_source_time()
+        self._validate_mode_object_bends()
+
+    def _validate_mode_object_bends(self) -> None:
+        """Error if any mode sources or monitors with bends have a radius that is too small."""
+        for imnt, monitor in enumerate(self.monitors):
```

Comment:
not sure if it would make most sense to collect all `imnt` and `isrc` and do the validation in one go? displaying all of the offending indices?

Reply:
Hm it's not what we usually do I believe. Just error asap sounds fine to me.

Reply:
sounds good.